command/resource/client/grpc-config.go

```diff
@@ -4,40 +4,118 @@
 package client
 
 import (
+	"fmt"
 	"os"
+	"strconv"
+	"strings"
 )
 
 const (
 	// GRPCAddrEnvName defines an environment variable name which sets the gRPC
 	// server address for the consul CLI.
 	GRPCAddrEnvName = "CONSUL_GRPC_ADDR"
+
+	// GRPCTLSEnvName defines an environment variable name which sets the gRPC
+	// communication mode. Default is false in plaintext mode.
+	GRPCTLSEnvName = "CONSUL_GRPC_TLS"
+
+	// GRPCSSLVerifyEnvName defines an environment variable name which sets
+	// whether to disable certificate checking.
+	GRPCSSLVerifyEnvName = "CONSUL_GRPC_SSL_VERIFY"
```

Comment:
nit:

This would probably be better named `GRPCTLSVerifyEnvName` and have a value of `"CONSUL_GRPC_TLS_VERIFY"`


SSL is ancient terminology at this point but back when the HTTP API env var was added SSL was still at version 3.0 prior to changes to the protocol and renaming to TLS.